logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -1,7 +1,9 @@
 # encoding: utf-8
 
-require "jruby_acked_queue_ext"
-require "jruby_acked_batch_ext"
+# Force loading the RubyUtil to ensure its loaded before the WrappedAckedQueue class is set up in
+# Ruby since WrappedAckedQueue depends on Ruby classes that are dynamically set up by Java code.
+java_import org.logstash.RubyUtil
```

Comment:
This _feels_ fragile. It it possible to create a `java_support.rb` (or the like, with documentation) that can be required where ever this type of import is needed.

Reply:
@jakelandis you're 100% correct here imo (this is stable though, I promise :D it triggers classloading `RubyUtil` in 100% of cases). The thing is we're already doing exactly this in `event.rb`, so I'd rather not mix in drying this up here.

**BUT** come to think of it, life might be even better here in the next step :) You know, I think we should simply put this line into just `environment.rb` and it will work 100% fine (provided https://github.com/elastic/logstash/pull/8594 is in).

It's probably not worth it cleaning this up now when https://github.com/elastic/logstash/pull/8594 is just around the corner and we can simply move this to `environment.rb` then right?

Reply:
I think this is the 3rd time I have seen same fix, and we probably don't want to rely on environment.rb since unit tests don't go through that path. 

No concerns about the technical correctness... just the human ability to connect the dots for needing to import RubyUtil.  My suggestion is basically to create shim ruby file that basically does nothing but call this `java_import` and has some documentation for when to use it. 

It's just my opinion on readability and don't want to block progress.

Reply:
@jakelandis I think the `rspec` path also goes through `environment.rb`, this is what's in there:

rspec.rb
```ruby
require_relative "environment"
LogStash::Bundler.setup!({:without => [:build]})
require "logstash-core"
require "logstash/environment"

$LOAD_PATH.unshift(File.join(LogStash::Environment::LOGSTASH_CORE, "spec"))
```

?